lms/djangoapps/grades/migrations/0001_initial.py

```diff
@@ -2,19 +2,42 @@
 from __future__ import unicode_literals
```

Comment:
@jibsheet Can we get some devops eyes on this? Migrations 0001 and 0002 have been applied, but all the tables are empty as the feature has not yet been enabled. We need to add a column to a table, but since django thinks this table already has data, it won't let the column be added without a default. Would this plan of action be possible?

- Rollback all migrations to the `grades` djangoapp using currently deployed AMI
- Deploy new AMI, migrate the new 0001 as usual